commit 1d2853c1ec4a1375d011b295c210e5220c1fda90
Author: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date:   Fri Jan 2 12:05:39 2015 +0200

    Make vtkfox compatible with CMake 3.1.0.

    Use VTK_USE_FILE instead of USE_VTK_FILE. The latter had been deprecated
    for 12 years and was only present in CMake's FindVTK.cmake, which was
    removed in 3.1.0 because VTK ships its own Config.cmake file with all
    the required variables.

--- CMakeLists.txt
+++ CMakeLists.txt
@@ -8,7 +8,7 @@ SET (VTKFOX_VERSION 1.0.0)
 # VTK
 #
 FIND_PACKAGE(VTK REQUIRED)
-INCLUDE(${USE_VTK_FILE})
+INCLUDE(${VTK_USE_FILE})
 
 #
 # FOX version to use
